airbyte-webapp/src/pages/connections/ConnectionPage/ConnectionPageTitle.tsx

```diff
@@ -26,7 +26,8 @@ export const ConnectionPageTitle: React.FC = () => {
 
   const { connection } = useConnectionEditService();
 
-  const { data: displayEnrollmentCallout } = useFreeConnectorProgram();
+  const { data: freeConnectorProgramInfo } = useFreeConnectorProgram();
+  const displayEnrollmentCallout = freeConnectorProgramInfo?.showEnrollmentUi;
```

Comment:
This can probably be the same in both `CreditsPage` and `ConnectionTitlePage` if we just rename the variable `showEnrollmentUi`...  No real strong feeling in how we do it.